Fixed `recipe_impact.yml`

schlunma opened the pull request:
<!--
    Thank you for contributing to our project!

    Please do not delete this text completely, but read the text below and keep
    items that seem relevant. If in doubt, just keep everything and add your
    own text at the top, a reviewer will update the checklist for you.

    While the checklist is intended to be filled in by the technical and scientific
    reviewers, it is the responsibility of the author of the pull request to make
    sure all items on it are properly implemented.
-->

## Description

This PR removes a problematic dataset of `recipe_impact.yml` that caused the recipe to fails because of a messed up time coordinate.

## Checklist

It is the responsibility of the author to make sure the pull request is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.

<!-- The next two lines turn the 🛠 and 🧪 below into hyperlinks -->
[1]: https://docs.esmvaltool.org/en/latest/community/review.html#technical-review
[2]: https://docs.esmvaltool.org/en/latest/community/review.html#scientific-review

- [x] [🛠][1] This pull request has a [descriptive title](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#pull-request-title)
- [x] [🛠][1] Code is written according to the [code quality guidelines](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#code-quality)
- [x] [🛠][1] [Documentation](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#documentation) is available
- [x] [🛠][1] [Tests](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#tests) run successfully
- [x] [🛠][1] The [list of authors](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#list-of-authors) is up to date
- [x] [🛠][1] Any changed dependencies have been [added or removed correctly](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#dependencies)
- [x] [🛠][1] All [checks below this pull request](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#pull-request-checks) were successful

### [New or updated recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html)

- [x] [🧪][2] [Recipe runs successfully](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#testing-recipes)
- [x] [🧪][2] [Recipe is well documented](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recipe-and-diagnostic-documentation)
- [x] [🧪][2] [Figure(s) and data](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#diagnostic-output) look as expected from literature
- [x] [🛠][1] [Provenance information](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance) has been added

***

To help with the number of pull requests:

-  🙏 We kindly ask you to [review](https://docs.esmvaltool.org/en/latest/community/review.html#review-of-pull-requests) two other [open pull requests](https://github.com/ESMValGroup/ESMValTool/pulls) in this repository

<!--
If you need help with any of the items on the checklists above, please do not hesitate to ask by commenting in the issue or pull request.
-->

remi-kazeroni commented:
I don't think it makes sense to trigger the bot here because we have not turned on the automatic data download there and it seems to me that a lot of data are missing to run that recipe at DKRZ. The [main_log_debug.txt](https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0-test-rc3/recipe_impact_20220223_103317/run/main_log_debug.txt) obtained during the last round of recipe testing shows the problem that is fixed here:
```bash
esmvalcore.cmor.check.CMORCheckError: There were errors in variable tas:
 Time coordinate for var time is not monotonic
 time: Frequency mon does not match input data
in cube:
air_temperature / (K)               (-- : 48; latitude: 160; longitude: 320)
    Dimension coordinates:
        latitude                        -             x               -
        longitude                       -             -               x
    Auxiliary coordinates:
        time                            x             -               -
    Scalar coordinates:
        height                      2.0 m
    Cell methods:
        mean                        time (3 hours)
    Attributes:
        Conventions                 'CF-1.4'
        associated_files            'baseURL: http://cmip-pcmdi.llnl.gov/CMIP5/dataLocation gridspecFile: gridspec_atmos_fx_EC-EARTH_rcp85_r0i0p0.nc...
        branch_time                 2006.0
        cmor_version                '2.8.0'
        contact                     'Alastair McKinstry <alastair.mckinstry@ichec.ie>'
        experiment                  'RCP8.5'
        experiment_id               'rcp85'
        forcing                     'Nat,Ant'
        frequency                   'mon'
        initialization_method       1
        institute_id                'ICHEC'
        institution                 'EC-Earth (European Earth System Model)'
        model_id                    'EC-EARTH'
        modeling_realm              'atmos'
        nco_openmp_thread_number    1
        original_name               '2T'
        parent_experiment           'pre-industrial control'
        parent_experiment_id        'piControl'
        parent_experiment_rip       'r1i1p1'
        physics_version             1
        product                     'output'
        project_id                  'CMIP5'
        realization                 6
        references                  'Model described by Hazeleger et al. (Bull. Amer. Meteor. Soc., 2010, 91,...
        source                      'EC-EARTH 2.3 (2011); atmosphere: IFS (cy31R1+modifications, T159L62);...
        source_file                 '/mnt/lustre01/work/kd0956/CMIP5/data/cmip5/output1/ICHEC/EC-EARTH/rcp...
        table_id                    'Table Amon (26 July 2011) b26379e76858ab98b927917878a63d01'
        title                       'EC-EARTH model output prepared for CMIP5 RCP8.5'
loaded from file /mnt/lustre01/work/kd0956/CMIP5/data/cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas/tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-200912.nc```

remi-kazeroni reviewed (dismissed):
Looks fine to me, thanks! Since we can't run the recipe with the bot, we will have a closer look at this run during the next round of recipe testing.

bouweandela commented:
The recipe worked fine last time i ran it, the output is here: https://esmvaltool.dkrz.de/shared/esmvaltool/climate4impact/recipe_output.html. I do remember that i had to run it on my laptop and download all data, because on mistral there were some outdated models and the automatic download is not yet smart enough to check if a newer version is available even if the data is found locally.

schlunma commented:
I just checked the data, and indeed it looks weird:

```bash
 > ll `dirname /mnt/lustre01/work/kd0956/CMIP5/data/cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas/tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-205012.nc`                                                                                        30ms
total 456340
-rwxrwxr-x 1 k204210 esgf   9851620 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-200912.nc
-rw-rw-r-- 1 k204210 esgf 110624960 Jun 21  2012 tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-205012.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_201001-201912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_202001-202912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_203001-203912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_204001-204912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_205001-205912.nc
-rw-rw-r-- 1 k204210 esgf 122914400 Jun 21  2012 tas_Amon_EC-EARTH_rcp85_r6i1p1_205101-210012.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_206001-206912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_207001-207912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_208001-208912.nc
-rwxrwxr-x 1 k204210 esgf  24599348 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_209001-209912.nc
-rwxrwxr-x 1 k204210 esgf   2477960 Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_210001-210012.nc
```

The problem does not appear in the `tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-205012.nc` file. Maybe we should ask DKRZ to delete the duplicated data?

bouweandela commented:
It looks like this is a data issue on Mistral, not a problem with this recipe. Shall we close this pull request?

schlunma commented:
We just ran this recipe with data freshly downloaded from ESGF. Unfortunately, the error persists, so it's probably not an error on mistral. I suggest removing the erroneous dataset.

bouweandela reviewed (changes requested):
The problem is reported with the CMIP5 model, while this pull request removes a similar looking CMIP6 model

schlunma commented:
> The problem is reported with the CMIP5 model, while this pull request removes a similar looking CMIP6 model

Right, sorry about that!! Will fix that locally and test the recipe.

> Maybe we should consider making the code for selecting the most recent version of the data more advanced. But the fact that data can be stored over multiple versions as reported in https://github.com/ESMValGroup/ESMValCore/issues/286 makes this non-trivial.

Agreed! Can you open an issue about that? Would be nice to tackle this in v2.6.

schlunma commented:
I added other very similar changes to this PR. Here is a list of successful runs for the corresponding recipes:

https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0-test-rc4/recipe_perfmetrics_CMIP5_20220304_111729/
https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0-test-rc4/recipe_perfmetrics_CMIP5_4cds_20220304_114547/
https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0-test-rc4/recipe_impact_20220304_112324/
https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0-test-rc4/recipe_mpqb_xch4_20220304_092520/

remi-kazeroni commented:
The `recipe_landcover` runs fine with if the missing data listed in #2408 are removed. Note that with the updated version of esgf-pyclient, we are _very_ confident that the data are not available on ESGF nodes for the recipes of this PR.

remi-kazeroni commented on esmvaltool/recipes/recipe_landcover.yml:
```diff
@@ -29,7 +29,7 @@ documentation:
 datasets:
   - {dataset: MPI-ESM-LR,       project: CMIP5, exp: rcp85,  ensemble: r1i1p1,  start_year: 2008,  end_year: 2012}
-  - {dataset: inmcm4,           project: CMIP5, exp: rcp85,  ensemble: r1i1p1,  start_year: 2008,  end_year: 2012}
+  #- {dataset: inmcm4,           project: CMIP5, exp: rcp85,  ensemble: r1i1p1,  start_year: 2008,  end_year: 2012}
```
I'm not the right person to decide. If the recipe was used for some publications, I think it would be better to comment out the faulty models (and keep that for reference) than just deleting them from the recipes.

schlunma replied:
@axel-lauer what do you think?

bouweandela commented:
Could you update the top post so it clear describes what is changed in which recipes and why? Are there issues in which the problems are reported? The problem with recipe_impact.yml appears to be a combination of faulty data organization at DKRZ and a limitation of ESMValCore https://github.com/ESMValGroup/ESMValCore/issues/1523. What about the other recipes?

schlunma commented:
> Could you update the top post so it clear describes what is changed in which recipes and why? Are there issues in which the problems are reported? The problem with recipe_impact.yml appears to be a combination of faulty data organization at DKRZ and a limitation of ESMValCore [ESMValGroup/ESMValCore#1523](https://github.com/ESMValGroup/ESMValCore/issues/1523). What about the other recipes?

Done.

bouweandela commented:
Thanks! The problem with the CMIP5 EC-EARTH r6i1p1 dataset is not a wrong time coordinate, but that we are trying to use two versions of the dataset at the same time and these apparently have a different time coordinate. This happens because ESMValCore is not able to select the right files (https://github.com/ESMValGroup/ESMValCore/issues/1523). Using just the latest version would probably solve the issue.

schlunma commented:
> Thanks! The problem with the CMIP5 EC-EARTH r6i1p1 dataset is not a wrong time coordinate, but that we are trying to use two versions of the dataset at the same time and these apparently have a different time coordinate. This happens because ESMValCore is not able to select the right files ([ESMValGroup/ESMValCore#1523](https://github.com/ESMValGroup/ESMValCore/issues/1523)). Using just the latest version would probably solve the issue.

There is definitely an issue in the time coordinate in the old version of this:

```bash
> ncdump -v time tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-200912.nc                                                                                                                                                                                                2268ms
netcdf tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-200912 {
dimensions:
        time = UNLIMITED ; // (48 currently)
        lat = 160 ;
        lon = 320 ;
        bnds = 2 ;
variables:
        double time(time) ;
                time:bounds = "time_bnds" ;
                time:units = "days since 2006-1-1" ;
                time:calendar = "gregorian" ;
                time:axis = "T" ;
                time:long_name = "time" ;
                time:standard_name = "time" ;
        double time_bnds(time, bnds) ;
        double lat(lat) ;
                lat:bounds = "lat_bnds" ;
                lat:units = "degrees_north" ;
                lat:axis = "Y" ;
                lat:long_name = "latitude" ;
                lat:standard_name = "latitude" ;
        double lat_bnds(lat, bnds) ;
        double lon(lon) ;
                lon:bounds = "lon_bnds" ;
                lon:units = "degrees_east" ;
                lon:axis = "X" ;
                lon:long_name = "longitude" ;
                lon:standard_name = "longitude" ;
        double lon_bnds(lon, bnds) ;
        double height ;
                height:units = "m" ;
                height:axis = "Z" ;
                height:positive = "up" ;
                height:long_name = "height" ;
                height:standard_name = "height" ;
        float tas(time, lat, lon) ;
                tas:standard_name = "air_temperature" ;
                tas:long_name = "Near-Surface Air Temperature" ;
                tas:units = "K" ;
                tas:original_name = "2T" ;
                tas:cell_methods = "time: mean (interval: 3 hours)" ;
                tas:cell_measures = "area: areacella" ;
                tas:history = "2013-01-13T11:10:30Z altered by CMOR: Treated scalar dimension: \'height\'. 2013-01-13T11:10:30Z altered by CMOR: replaced missing value flag (1e+28) with standard missing value (1e+20). 2013-01-13T11:10:30Z altered by CMOR: Converted type
 from \'d\' to \'f\'. 2013-01-13T11:10:30Z altered by CMOR: Inverted axis: lat." ;
                tas:coordinates = "height" ;
                tas:missing_value = 1.e+20f ;
                tas:_FillValue = 1.e+20f ;
                tas:associated_files = "baseURL: http://cmip-pcmdi.llnl.gov/CMIP5/dataLocation gridspecFile: gridspec_atmos_fx_EC-EARTH_rcp85_r0i0p0.nc areacella: areacella_fx_EC-EARTH_rcp85_r0i0p0.nc" ;

// global attributes:
                ...
data:

 time = 15.5, 45, 74.5, 105, 135.5, 166, 196.5, 227.5, 258, 288.5, 319, 
    349.5, 15.5, 45, 74.5, 105, 135.5, 166, 196.5, 227.5, 258, 288.5, 319, 
    349.5, 15.5, 45.5, 75.5, 106, 136.5, 167, 197.5, 228.5, 259, 289.5, 320, 
    350.5, 15.5, 45, 74.5, 105, 135.5, 166, 196.5, 227.5, 258, 288.5, 319, 
    349.5 ;
```

But you are right, using only the latest version would most likely fix this.

bouweandela commented:
Thanks for clarifying!

remi-kazeroni commented:
> Thanks! The problem with the CMIP5 EC-EARTH r6i1p1 dataset is not a wrong time coordinate, but that we are trying to use two versions of the dataset at the same time and these apparently have a different time coordinate. This happens because ESMValCore is not able to select the right files ([ESMValGroup/ESMValCore#1523](https://github.com/ESMValGroup/ESMValCore/issues/1523)). Using just the latest version would probably solve the issue.

Taking a closer look at the data available on Mistral, it looks to me that the newest of the two data versions available show a problem with the time coordinate:

```bash
ll /mnt/lustre01/work/kd0956/CMIP5/data/cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas
total 446M
-rw-rw-r--  1 k204210 esgf 106M Jun 21  2012 tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-205012.nc
-rw-rw-r--  1 k204210 esgf 118M Jun 21  2012 tas_Amon_EC-EARTH_rcp85_r6i1p1_205101-210012.nc
-rwxrwxr-x  1 k204210 esgf 9.4M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-200912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_209001-209912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_207001-207912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_204001-204912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_202001-202912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_201001-201912.nc
-rwxrwxr-x  1 k204210 esgf 2.4M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_210001-210012.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_208001-208912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_206001-206912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_205001-205912.nc
-rwxrwxr-x  1 k204210 esgf  24M Dec  1  2017 tas_Amon_EC-EARTH_rcp85_r6i1p1_203001-203912.nc
```
and the weird time coordinate is only in the model from Dec 1 2017. Nevertheless the data looks fine if downloaded from ESGF nodes. 

From a testing perspective, the recipe_impact is challenging to run. Using Mistral data or downloaded data only is not sufficient to get a successful run. Given that DKRZ does not really support CMIP5 data any more, I am not sure they would clean that directory and download the latest data. Not sure that is a good enough reason for commenting out the faulty dataset from the recipe. But if we keep problems like this in our recipes, testing them automatically will remain challenging, I have seen that...

schlunma commented:
That's weird. From the screenshot of @bouweandela it looks like the problematic file is the old version (`v20130315`).
 
> ```python
> >>> from esmvalcore.esgf import find_files
> >>> files = find_files(project="CMIP5", mip="Amon", short_name="tas", dataset='EC-EARTH', exp=['historical', 'rcp85'], ensemble='r6i1p1')
> >>> for file in files:
> ...     print(file)
> ... 
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/historical/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_historical_r6i1p1_190001-194912.nc on hosts ['aims3.llnl.gov', 'esgf-data1.ceda.ac.uk', 'esgf.nci.org.au', 'esgf2.dkrz.de']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/historical/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_historical_r6i1p1_195001-200512.nc on hosts ['aims3.llnl.gov', 'esgf-data1.ceda.ac.uk', 'esgf.nci.org.au', 'esgf2.dkrz.de']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-200912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20171115/tas_Amon_EC-EARTH_rcp85_r6i1p1_200601-205012.nc on hosts ['esgf.ichec.ie', 'esgf.nci.org.au']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_201001-201912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_202001-202912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_203001-203912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_204001-204912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_205001-205912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20171115/tas_Amon_EC-EARTH_rcp85_r6i1p1_205101-210012.nc on hosts ['esgf.ichec.ie', 'esgf.nci.org.au']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_206001-206912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_207001-207912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_208001-208912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_209001-209912.nc on hosts ['aims3.llnl.gov']
> ESGFFile:cmip5/output1/ICHEC/EC-EARTH/rcp85/mon/atmos/Amon/r6i1p1/v20130315/tas_Amon_EC-EARTH_rcp85_r6i1p1_210001-210012.nc on hosts ['aims3.llnl.gov']
> >>> 
> ```

I'm really in favor for dropping this dataset. The recipe does not really depend on it and it caused already so much trouble during the tests.

remi-kazeroni commented:
I have run successfully the 5 recipes of this PR. The overview of the results can be found in this [temporary table](https://esmvaltool.dkrz.de/shared/esmvaltool/v2.5.0-test-rc4_2563/). Once this is merged, I will update the v2.5.0_rc4 table accordingly. Note that I used a dev installation of the Core to apply the latest fix for the perfmetrics recipes (https://github.com/ESMValGroup/ESMValCore/pull/1524)

schlunma commented:
@bouweandela can you remove your "requested changes" review if you are happy with this?

@axel-lauer What is your opinion on how to deal with missing datasets? Commenting them or removing them?

axel-lauer commented:
> @axel-lauer What is your opinion on how to deal with missing datasets? Commenting them or removing them?

If in doubt, I would prefer to simply comment out things. This way, we still have some kind of "tracking" as the recipe does no longer reproduce the original figure. At least, then we do not forget as easily.

schlunma commented:
This should be ready now
